fastlane/lib/fastlane/actions/commit_github_file.rb

```diff
@@ -0,0 +1,184 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      COMMIT_GITHUB_FILE_HTML_LINK = :COMMIT_GITHUB_FILE_HTML_LINK
+      COMMIT_GITHUB_FILE_SHA = :COMMIT_GITHUB_FILE_SHA
+      COMMIT_GITHUB_FILE_JSON = :COMMIT_GITHUB_FILE_JSON
+    end
+
+    class CommitGithubFileAction < Action
+      def self.run(params)
+        repo_name = params[:repository_name]
+        branch = params[:branch] ||= 'master'
+        commit_message = params[:message]
+
+        file_path = params[:path]
+        file_name = File.basename(file_path)
+        expanded_file_path = File.expand_path(file_path)
+
+        UI.important("Creating commit on #{repo_name} on branch \"#{branch}\" for file \"#{file_path}\"")
```

Comment:
Curious to hear your thoughts about making some of these important and message lines use `UI.verbose` instead to keep noise down unless the user asks for it?